modules/oe_content_event/src/EventNodeWrapper.php

```diff
@@ -97,6 +97,27 @@ public function getEndDate(): DrupalDateTime {
     return $this->entity->get('oe_event_dates')->end_date;
   }
 
+  /**
```

Comment:
Methods should be tested in modules/oe_content_event/tests/src/Kernel/EventNodeWrapperTest.php